demos/tabs.scss

```diff
@@ -57,3 +58,11 @@
     @include mdc-tab-ink-color($material-color-indigo-500);
   }
 }
+
+.custom-indicator-tab-bar {
+  @include mdc-tab-bar-indicator-ink-color(primary);
+}
+
+.custom-toolbar-indicator .custom-indicator-toolbar-tab-bar {
+  @include mdc-tab-bar-indicator-ink-color(secondary);
```

Comment:
this example seems to do the same thing as above...do we really need it?

Reply:
Its similar, but a different color (primary vs secondary). Also we need it to target the toolbar specifier. If we don't have that we cannot override the color. If we remove the toolbar scss, we can probably get around this.

Reply:
I'll update this after I remove toolbar specific sass

Reply:
Addressed in https://github.com/material-components/material-components-web/pull/1979/